mappers/bluetooth_mapper/data_converter/data_converter.go

```diff
@@ -90,9 +91,7 @@ func (operation *ReadOperation) ConvertReadData(data []byte) float64 {
 			initialValue = append(initialValue, data[index])
 		}
 	}
-	for _, value := range initialValue {
-		initialStringValue = initialStringValue + strconv.Itoa(int(value))
-	}
+	initialStringValue = hex.EncodeToString(initialValue)
```

Comment:
Hi @dhdljd ,
Thanks for the change
Did u get a chance to test this change ? Does it produce the same value as the previously expected ?

Reply:
Could u pls test this change....I think the value produced will be different from what was previously returned....It would be helpful if u could verify this....

Reply:
You could take a look use this guide if u would like to test your changes through the e2e env for bluetooth: https://github.com/kubeedge/kubeedge/blob/master/docs/guides/bluetooth_mapper_e2e_guide.md